kitsune/settings.py

```diff
@@ -832,29 +834,60 @@ def JINJA_CONFIG():
 # Badges (kbadge)
 BADGE_IMAGE_PATH = "uploads/badges/"
 
-# Email
-EMAIL_BACKEND = config("EMAIL_BACKEND", default="kitsune.lib.email.LoggingEmailBackend")
-EMAIL_LOGGING_REAL_BACKEND = config(
-    "EMAIL_LOGGING_REAL_BACKEND", default="django.core.mail.backends.console.EmailBackend"
+# Email settings
+EMAIL_BACKEND = config("EMAIL_BACKEND", default="post_office.EmailBackend")
+POST_OFFICE_DEFAULT_EMAIL_BACKEND = config(
+    "POST_OFFICE_DEFAULT_EMAIL_BACKEND", default="django_ses.SESBackend"
 )
 EMAIL_SUBJECT_PREFIX = config("EMAIL_SUBJECT_PREFIX", default="[support] ")
-if EMAIL_LOGGING_REAL_BACKEND in (
+# What email backend is actually sending the emails?
+REAL_EMAIL_BACKEND = (
+    POST_OFFICE_DEFAULT_EMAIL_BACKEND
+    if EMAIL_BACKEND == "post_office.EmailBackend"
+    else EMAIL_BACKEND
+)
+# Settings for specific email backends.
+if REAL_EMAIL_BACKEND in (
     "bandit.backends.smtp.HijackSMTPBackend",
     "django.core.mail.backends.smtp.EmailBackend",
     "kitsune.lib.email.SMTPEmailBackendWithSentryCapture",
 ):
+    # Settings for SMTP email backends.
     EMAIL_HOST = config("EMAIL_HOST")
     EMAIL_HOST_USER = config("EMAIL_HOST_USER")
     EMAIL_HOST_PASSWORD = config("EMAIL_HOST_PASSWORD")
     EMAIL_PORT = config("EMAIL_PORT", default=25, cast=int)
     EMAIL_USE_TLS = config("EMAIL_USE_TLS", default=False, cast=bool)
-elif EMAIL_LOGGING_REAL_BACKEND == "django_ses.SESBackend":
+elif REAL_EMAIL_BACKEND == "django_ses.SESBackend":
+    # Settings for the django_ses.SESBackend.
     AWS_SES_ACCESS_KEY_ID = config("AWS_SES_ACCESS_KEY_ID")
     AWS_SES_SECRET_ACCESS_KEY = config("AWS_SES_SECRET_ACCESS_KEY")
     AWS_SES_REGION_NAME = config("AWS_SES_REGION_NAME", default="us-west-2")
     AWS_SES_AUTO_THROTTLE = None
     USE_SES_V2 = config("USE_SES_V2", default=True, cast=bool)
 
+# Settings for django-post-office.
+POST_OFFICE = {
+    "CELERY_ENABLED": True,
+    "BACKENDS": {
+        "default": POST_OFFICE_DEFAULT_EMAIL_BACKEND,
+    },
+    "THREADS_PER_PROCESS": 1,
+    "BATCH_SIZE": 10,
```

Comment:
Let's start with a higher number here. I doubt all the workers will fire at the same time, a batch size of 30 should be conservative enough